### PR TITLE
OPENSHIFTP-168: add symlink for openshift-install-fips

### PR DESCRIPTION
**Pull request by prb112**
OPENSHIFTP-168: add symlink for openshift-install-fips

### CODE REVIEW

File: vars/main.yml [hunk 1]
```diff
@@ -28,3 +28,4 @@ setup_registry:
   registry_user: "admin"
   registry_password: "admin"
 machineconfig_path: ../machineconfig
+fips: false
```
Comment (Prajyot-Parab) [inline vars/main.yml:31]:
```suggestion
fips: false

```

**Review by Prajyot-Parab — CHANGES_REQUESTED** [review]
please squash commits @prb112

**Comment by prb112** [comment]
Hey Prajyot,
Squashed the four commits. 
Back to you, thanks, Paul

**Comment by prb112** [comment]
Thank you

**Pull request merged**